app/src/main/java/io/ooni/mk/nuvolari/android/MainActivity.java

```diff
@@ -0,0 +1,68 @@
+package io.ooni.mk.nuvolari.android;
+
+import android.os.Handler;
+import android.os.Looper;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.util.Log;
+import android.widget.Toast;
+
+import io.ooni.mk.nuvolari.libndt7.Client;
+import io.ooni.mk.nuvolari.libndt7.Measurement;
+import io.ooni.mk.nuvolari.libndt7.Settings;
+
+public class MainActivity extends AppCompatActivity {
+
+    @NonNull
+    private static final String TAG = "MainActivity";
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+
+        Settings settings = new Settings();
+        settings.hostname = "35.235.104.27";
+        settings.port = 443;
+        settings.disableTLS = false;
+        settings.skipTLSVerify = true;
+        settings.download.duration = 1;
+        settings.download.adaptive = true;
```

Comment:
This setting and also duration are now ignored by the server, so they should be removed.